### PR TITLE
Moving blog entries into a single `blog/` folder

### PR DESCRIPTION
**Pull request by JKAussieSkater**
As we gradually add more and more blog entries, it will become increasingly difficult to manage the grunt-docs, as most of the documentation's source files falls after the letter "b" (and we have to scroll heaps to find the files we need to edit). Organising blog entries into the same `blog/` folder could help manage the repository better.

## IMPORTANT
This pull request is tied to a pull request in another repository. Merging this pull request must follow a precise sequence of events:
- [ ] FIRST merge THIS pull request into `gruntjs/grunt-docs`, and note the latest commit code
- [ ] SECONDLY merge [the other pull request](https://github.com/gruntjs/gruntjs.com/pull/168) into the `gruntjs/gruntjs.com` repository
- [ ] Update the [`gruntjs/gruntjs.com/package.json`](https://github.com/gruntjs/gruntjs.com/blob/master/package.json) file in the other repository so the "grunt-docs" URL has the latest commit code (noted from the first checkbox)

### CODE REVIEW

**Comment by shama** [comment]
This looks like a good idea to me. @vladikoff ?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (vladikoff) [inline .gitignore]: Undo this change, setup a global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
